### DISCUSSION TITLE
Add override for aliases that need forwarding to 3rd party apps and services

### DISCUSSION THREAD

**Discussion by gtmadev** [simple-login/app#522] [Feature Requests]
There should be an option for aliases where you can force it to forward as the original email. 

Why would we want this? Think of a helpdesk app that needs us to forward emails to support@ to the service. With the current service, this can't work because the customer's email is stripped out and can't match to a real customer in the third party app.

To get around this problem, I've been forced to create a dedicated domain for support so that it can be setup with a real email address that can forward the email with customer's email intact. But this is rather confusing for customers because now our domain for support does not match our main domain.

**Comment by nguyenkims** [comment]
Currently SimpleLogin sets the header `X-SimpleLogin-Envelope-From` to the original sender, I wonder if so maybe you can set up the Helpdesk to use this header instead of the From: header?

**Reply by gtmadev** [reply]
Unfortunately they don't provide that granular of control. They do parse parts of the header to see if the email was forwarded from a regular mail client - and that does work.

What I've done for now is I use a different domain for support queues and I don't route those through SimpleLogin. I can still route the old address through SimpleLogin to the helpdesk, but it will come in as a new user and not to the existing customer - which is matched by the customer's real email address. I will just have to manually change those if anyone still sends to the old support address.

But the ideal thing here would be to have a little checkbox on aliases that says override and forward as regular email - or something like that. All it would do is just bypass the reverse alias part so that we can have exceptions such as the one I have documented in this feature request.

Thanks for your consideration.

**Reply by nguyenkims** [reply]
I see, in this case I think you can subscribe to this thread https://github.com/simple-login/app/discussions/504 which is about to support ARC which will keep the original sender address.
